controlplane/eks/api/v1alpha3/awsmanagedcontrolplane_types.go

```diff
@@ -166,6 +166,8 @@ type EncryptionConfig struct {
 type OIDCProviderStatus struct {
 	// ARN holds the ARN of the provider
 	ARN string `json:"arn,omitempty"`
+	// TrustPolicy contains an example trust policy to use with IAM roles
```

Comment:
Not sure this description makes sense

Reply:
I wonder if we need an option to choose whether the trust policy is stored as a configmap on the management or tenant cluster. Could be a flag to the manager instead exposing it via the api